src/eventConsumer/prospectEvents/prospectEventConsumer.spec.ts

```diff
@@ -0,0 +1,22 @@
+import { getProspectEventPayload } from './prospectEventConsumer';
+import { ProspectEventPayloadSnowplow } from '../../snowplow/prospect/types';
+import { testProspectData } from '../../snowplow/prospect/testData';
+
+describe('getProspectEventPayload', () => {
+  it('should convert request body to Prospect', () => {
+    const expected: ProspectEventPayloadSnowplow = {
+      object_version: 'new',
+      prospect: testProspectData,
+      eventType: 'PROSPECT_REVIEWED',
+    };
+
+    const requestBody = {
+      'detail-type': 'dismiss-prospect',
```

Comment:
we've changed this to `prospect-dismiss` now

Reply:
This name is so much better! 👍